composer.json

```
@@ -0,0 +1,14 @@
+{
+    "name": "wordpress/plugin-check",
+    "description": "WordPress Plugin Check Plugin",
+    "type": "wordpress-plugin",
+    "license": "GPL-2.0-or-later",
+    "require": {
+        "php": ">=5.6"
+    },
+    "autoload": {
+        "psr-4": {
+            "Wordpress\\Plugin_Check\\": "includes/"
```

```suggestion
            "WordPress\\Plugin_Check\\": "includes/"
```
Let's adjust that throughout :)

---

Thanks @felixarntz this has been updated and was the only instance I've found. I will keep an eye on this in future commits.